src/admin/components/Application/Application.tsx

```diff
@@ -0,0 +1,68 @@
+import Card from "react-bootstrap/Card";
+
+import { ApplicantActions, ApplicantStatus } from "admin/components";
+import {
+	Applicant,
+	ApplicationQuestion,
+	submitReview,
+} from "admin/utils/useApplicants";
+
+import ApplicationReviews from "./ApplicationReviews";
+import ApplicationSection from "./ApplicationSection/ApplicationSection";
+
+import styles from "./Application.module.scss";
+
+interface ApplicationSections {
+	[key: string]: ApplicationQuestion[];
+}
+
+const APPLICATION_SECTIONS: ApplicationSections = {
+	"Personal Information": ["gender", "pronouns", "ethnicity", "is_18_older"],
```

Comment:
Clarification: Are we missing email? I saw that first and last name were extracted, but I didn't see email.

Reply:
Initially didn't think it was that important to display email, but do you think we should?

Reply:
In general, I think having quick lookup to contact info would be useful, but since applications is running their own spreadsheets this isn't an issue.